### PR TITLE
[FE]: Customer Navbar

### PR DESCRIPTION
**Pull request by wpiyaphon**

### CODE REVIEW

File: src/_iconsbundle.json
Comment (panlabdev) [inline src/_iconsbundle.json:1]: what is this and why is it just in src

File: public/logo-white.png
Comment (panlabdev) [inline public/logo-white.png:1]: So this one cannot be svg right?

File: src/app/layout.tsx [hunk 1]
```diff
@@ -33,8 +33,11 @@ export default async function RootLayout({
         <meta charSet="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
       </head>
-      <body className={ibm.className}>
+      <body className={`${ibm.className} text-primaryT-darker`}>
         <Providers>
+          <nav>
```
Comment (panlabdev) [inline src/app/layout.tsx]: Do you really have to put inside another <nav> tag?

**Review by panlabdev — CHANGES_REQUESTED** [review]

File: src/_iconsbundle.json
Comment (wpiyaphon) [reply src/_iconsbundle.json:1]: forget to remove it, my attempts to render iconify on server side

File: public/logo-white.png
Comment (wpiyaphon) [reply public/logo-white.png:1]: there are only 2 variants, white and color. I don't think we need svg. Also, I tried to convert to svg but there's some defect on svg pixel

File: src/app/layout.tsx (hunk 1 above)
Comment (wpiyaphon) [reply src/app/layout.tsx]: yes, it's semantic tag for SEO performance

Comment (wpiyaphon) [reply src/app/layout.tsx]: actually nah, nextui component handles this

**Review by panlabdev — APPROVED** [review]

**Pull request merged**